style.css

```diff
@@ -80,4 +82,143 @@
     flex-direction: row;
     list-style-type: none;
   }
+
+  .works {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-content: flex-end;
+    align-items: center;
+    padding: 4rem;
+  }
+
+  .recent {
+    width: 327px;
+    height: 83px;
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+  }
+
+  .gridcontainer1 {
+    display: grid;
+  }
+
+  .stories {
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    color: #091e42;
+  }
+
+  .selection {
+    font-family: 'Inter', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #091e42;
+  }
+
+  .listbutton {
+    list-style: none;
+    display: flex;
+    gap: 12px;
+    margin-top: 1rem;
+  }
+
+  .button1,
+  .button2,
+  .button3,
+  .button4 {
+    border: 1px solid #8993a4;
+    padding: 10px 12px;
+  }
+
+  .button5 {
+    background: #ff6b00;
+    padding: 12px;
+    margin-top: 12px;
+    width: 124px;
+    height: 48px;
+    color: #fff;
+  }
+
+  .listproject {
+    background-image: url("secondimage.png");
+    background-repeat: no-repeat;
+    margin-top: 2rem;
+    height: 50vh;
+  }
+
+  .art {
+    margin: 2rem;
+    font-size: 2rem;
+    font-weight: 400;
+  }
+
+  .daily,
+  .art {
+    color: #ffff;
+  }
+
+  .daily {
+    font-family: 'Inter', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    margin: 2rem;
+  }
+
+  .button6,
+  .button7,
+  .button8 {
+    background: rgba(255, 255, 255, 0.24);
+    padding: 10px 12px;
+    color: #fff;
+  }
+
+  .listbutton2 {
+    gap: 12px;
+    display: flex;
+    margin-top: 2rem;
+    margin-left: 2rem;
+    margin-right: 2rem;
+    list-style: none;
+  }
+
+  .button9 {
+    /* height: auto; */
+    background: #ff6b00;
+    padding: 12px;
+    width: 100%;
+    height: 48px;
+    color: #fff;
+    margin-bottom: 1rem;
+  }
+
```

Comment:
Please kindly implement the active, pressed and hover effects of all the buttons. It is true that these effects are not seen on mobile versions but it is a must to implement them at this stage of the project. (_check the last project bulleted point in the image below_).You can check the desktop side of the figma design for the button effects. :pray:

![Screenshot from 2022-09-23 16-24-44](https://user-images.githubusercontent.com/65233860/192027091-73092812-c1e4-4bca-a444-fddd658be04c.png)